### ISSUE TITLE
Merged pathway still on new site

### ISSUE THREAD

**Issue by larsgw** [wikipathways/wikipathways-database#20]
Compare:
  - https://www.wikipathways.org/index.php/Pathway:WP3963 [deleted after merge; Sept 2022]
  - https://new.wikipathways.org/pathways/WP3963.html

**Comment by egonw** [comment]
The interesting thing here, is that WP3963 actually is cited in literature at least twice. How will we handle this?

**Comment by AlexanderPico** [comment]
Interesting case. I reviewed the deletion log. It was deleted for effectively being merged with https://new.wikipathways.org/pathways/WP4190. But it's clearly still distinct in terms of scope, so I restored it and marked it as approved.

WP3963 still exists in both wp-db and the jekyll site, so I think this won't have much affect (maybe a minor gpml-triggered update at most).

Resolved?

_Disclaimer: I'm not suggesting that this should always be the solution. Future cases of deleting approved and cited pathways should be handled case-by-case._
